src/Spryker/Zed/SynchronizationBehavior/Persistence/Propel/Behavior/SynchronizationBehavior.php

```diff
@@ -168,6 +171,19 @@ public function modifyTable()
             $uniqueIndex->addColumn($table->getColumn('key'));
             $table->addUnique($uniqueIndex);
         }
+
+        if (isset($parameters['mapping_resource'])) {
```

Comment:
Can you move this next to other `if (isset($parameters... {` parts, it's more consistent